exasol_sagemaker_extension/resources/create_statement_autopilot_endpoint_deployment_udf.sql

```diff
@@ -5,7 +5,7 @@ CREATE OR REPLACE PYTHON3_SME SET SCRIPT "SME_AUTOPILOT_ENDPOINT_DEPLOYMENT_UDF"
     instance_count INTEGER,
     aws_s3_connection VARCHAR(50),
     aws_region VARCHAR(20)
-) EMITS (endpoint_name VARCHAR(32)) AS
+) EMITS (endpoint_problem_type VARCHAR(32)) AS
```

Comment:
I guess, we don't need the endpoint name as a return value, because we don't change it in the UDF itself.

Reply:
It returns 'endpoint_problem_type' so that we are able to declare the  PredictoinUDF's output type